src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public enum Color {
+    White,
+    Red,
+    Orange,
+    Yellow,
+    Green,
+    Blue,
+    Violet
```

Comment:
Enum values should be in uppercase letters. Please change the enum values to follow the naming conventions for constants.